meson.build

```
@@ -0,0 +1,253 @@
+project('pysimbi', 'c', 'cpp', 'cython',
+    version : run_command('python', 'pysimbi/version.py', check: true).stdout().strip(), 
```

I notice that this version.py script simply reads out `pysimbi/VERSION`. Meson supports this as a builtin thing.

```suggestion
    version: files('pysimbi/VERSION'),
```

This avoids running an external process (which may not exist, if some distro refuses to make the `python` command a symlink to python3 -- some distros insist that because it historically used to be python2, it's "wrong" to make it be python3, so they prefer to let people get "error: command not found" instead).

It also means that Meson will reconfigure if it detects that the VERSION file was updated, and the build should be for a new version.


If you do this, you should also add `meson_version: '>=0.57.0'` to make sure that no one uses too old versions of python where this doesn't work.